# HG changeset patch
# User Rik <rik@octave.org>
# Date 1354984023 28800
#      Sat Dec 08 08:27:03 2012 -0800
# Node ID 3216dafbbd48bd77d367d328b8bfdc0ebc486e2e
# Parent  bf77f7f66bdfa86f47ca1f5d2850a83e1b643211
ov-java.cc: Correcty typo HAVE_WINDWOS_H to HAVE_WINDOWS_H

* ov-java.cc: Correcty typo HAVE_WINDWOS_H to HAVE_WINDOWS_H

diff --git a/libinterp/octave-value/ov-java.cc b/libinterp/octave-value/ov-java.cc
--- a/libinterp/octave-value/ov-java.cc
+++ b/libinterp/octave-value/ov-java.cc
@@ -21,17 +21,17 @@ along with Octave; see the file COPYING.
 */
 
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #if defined HAVE_JAVA
 
-#if defined (HAVE_WINDWOS_H)
+#if defined (HAVE_WINDOWS_H)
 #include <windows.h>
 #endif
 
 #include <algorithm>
 #include <map>
 #include <iostream>
 #include <fstream>
 
